webrender/src/renderer.rs

```diff
@@ -88,6 +88,12 @@ const GPU_TAG_PRIM_BOX_SHADOW: GpuProfileTag = GpuProfileTag { label: "BoxShadow
 // Orange
 const GPU_TAG_PRIM_BORDER: GpuProfileTag = GpuProfileTag { label: "Border", color: ColorF { r: 1.0, g: 0.5, b: 0.0, a: 1.0 } };
 
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum BlendMode {
+    None,
```

Comment:
Isn't this non-idiomatic? I'd think an `Option<BlendMode>` would be appropriate, where the actual `BlendMode` variants are all blend-enabled, even if the equation is trivial.